### PR TITLE
docs: update the usage of craco-less

### PR DESCRIPTION
**Pull request by hehex9**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Perfermance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | update the usage of craco-less |
| 🇨🇳 Chinese | 更新 craco-less 用法 |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered docs/react/use-with-create-react-app.en-US.md](https://github.com/hehex9/ant-design/blob/patch-1/docs/react/use-with-create-react-app.en-US.md)
[View rendered docs/react/use-with-create-react-app.zh-CN.md](https://github.com/hehex9/ant-design/blob/patch-1/docs/react/use-with-create-react-app.zh-CN.md)

### CODE REVIEW

File: docs/react/use-with-create-react-app.en-US.md [hunk 1]
```diff
@@ -146,8 +146,10 @@ module.exports = {
       plugin: CracoLessPlugin,
       options: {
         lessLoaderOptions: {
-          modifyVars: { '@primary-color': '#1DA57A' },
-          javascriptEnabled: true,
+          lessOptions: {
+            modifyVars: { '@primary-color': '#1DA57A' },
+            javascriptEnabled: true,
+          },
```
Comment (afc163) [inline docs/react/use-with-create-react-app.en-US.md:152]: Wrong usage, please check https://github.com/DocSpring/craco-less#configuration

image

Comment (hehex9) [reply docs/react/use-with-create-react-app.en-US.md:152]: check https://github.com/DocSpring/craco-less/pull/42

Comment (afc163) [reply docs/react/use-with-create-react-app.en-US.md:152]: You are right.

Comment (afc163) [reply docs/react/use-with-create-react-app.en-US.md:152]: We should update https://ant.design/docs/react/use-in-typescript too.

**Pull request merged**